Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM registry.access.redhat.com/ubi8/python-39@sha256:496f9eee1837022e749a5f2d5c9ae91720200e03c38528894aee4eb6b3bb78b5
+
+USER root
+
+RUN INSTALL_PKGS="flask ansible_runner ansible-pylibssh jsonschema GitPython requests" && \
+    pip install $INSTALL_PKGS
```

Comment:
No need to change this; I just wanted to comment that for package installation, writing it so that you have one package per line will make for easier to read diffs in the future:

```
RUN pip install \
  flask \
  ansible_runner \
  ansible-pylibssh \
  jsonschema \
  GitPython \
  requests
```